Yay crashes at the end (runtime error: makeslice: cap out of range)

#### Affected Version
yay v9.0.1 - libalpm v11.0.2

#### Issue
Yay crashes really rarely with message:
`panic: runtime error: makeslice: cap out of range`

<!-- The following sections may be left out if not relevant -->
#### Steps to reproduce
1. Simply update AUR packages

#### Output

```
 ✘ theaifam5@pussy-master  ~  yay -Syuu --devel
:: Synchronizing package databases...
 core is up to date
 extra is up to date
 community is up to date
 repo-ck is up to date
:: Starting full system upgrade...
 there is nothing to do
:: Searching databases for updates...
:: Searching AUR for updates...
:: Checking development packages...
:: 4 Packages to upgrade.
4  aur/clion          1:2018.3.2-1 -> 1:2018.3.3-1
3  aur/clion-gdb      1:2018.3.2-1 -> 1:2018.3.3-1
2  aur/clion-lldb     1:2018.3.2-1 -> 1:2018.3.3-1
1  aur/slack-desktop  3.3.3-1      -> 3.3.4-1
==> Packages to not upgrade: (eg: 1 2 3, 1-3, ^4 or repo name)
==> 
:: Checking for conflicts...
:: Checking for inner conflicts...
[Repo Make: 1]  rsync-3.1.3-1
[Aur: 4]  slack-desktop-3.3.4-1  clion-1:2018.3.3-1 (clion clion-gdb clion-lldb)

==> Remove make dependencies after install? [y/N] y
  2 slack-desktop                            (Installed) (Build Files Exist)
  1 clion (clion clion-gdb clion-lldb)       (Installed) (Build Files Exist)
==> Packages to cleanBuild?
==> [N]one [A]ll [Ab]ort [I]nstalled [No]tInstalled or (1 2 3, 1-3, ^4)
==> 
:: Downloaded PKGBUILD (1/2): slack-desktop
:: Downloaded PKGBUILD (2/2): clion (clion clion-gdb clion-lldb)
:: Parsing SRCINFO (1/2): slack-desktop
:: Parsing SRCINFO (2/2): clion (clion clion-gdb clion-lldb)
  2 slack-desktop                            (Installed) (Build Files Exist)
  1 clion (clion clion-gdb clion-lldb)       (Installed) (Build Files Exist)
==> [N]one [A]ll [Ab]ort [I]nstalled [No]tInstalled or (1 2 3, 1-3, ^4)
==> PKGBUILDs to edit?
==> 
:: Synchronizing package databases...
 core is up to date
 extra is up to date
 community is up to date
 repo-ck is up to date
resolving dependencies...
looking for conflicting packages...

Packages (1) rsync-3.1.3-1

Total Installed Size:  0.55 MiB

:: Proceed with installation? [Y/n] y
(1/1) checking keys in keyring                     [######################] 100%
(1/1) checking package integrity                   [######################] 100%
(1/1) loading package files                        [######################] 100%
(1/1) checking for file conflicts                  [######################] 100%
(1/1) checking available disk space                [######################] 100%
:: Processing package changes...
(1/1) installing rsync                             [######################] 100%
:: Running post-transaction hooks...
(1/2) Reloading system manager configuration...
(2/2) Arming ConditionNeedsUpdate...
==> Making package: slack-desktop 3.3.4-1 (Fri 11 Jan 2019 05:44:29 PM CET)
==> Retrieving sources...
  -> Downloading slack-desktop-3.3.4-amd64.deb...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 53.7M  100 53.7M    0     0  8422k      0  0:00:06  0:00:06 --:--:-- 8839k
  -> Found slack-desktop.patch
==> Validating source files with sha256sums...
    slack-desktop-3.3.4-amd64.deb ... Passed
    slack-desktop.patch ... Passed
==> Cleaning up...
==> Making package: clion 1:2018.3.3-1 (Fri 11 Jan 2019 05:44:37 PM CET)
==> Retrieving sources...
  -> Downloading CLion-2018.3.3.tar.gz...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   154  100   154    0     0    442      0 --:--:-- --:--:-- --:--:--   442
100  433M  100  433M    0     0   9.8M      0  0:00:44  0:00:44 --:--:--  9.9M
  -> Found jetbrains-clion.desktop
==> Validating source files with sha256sums...
    CLion-2018.3.3.tar.gz ... Passed
    jetbrains-clion.desktop ... Passed
==> Cleaning up...
==> Making package: slack-desktop 3.3.4-1 (Fri 11 Jan 2019 05:45:22 PM CET)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Found slack-desktop-3.3.4-amd64.deb
  -> Found slack-desktop.patch
==> Validating source files with sha256sums...
    slack-desktop-3.3.4-amd64.deb ... Passed
    slack-desktop.patch ... Passed
==> Removing existing $srcdir/ directory...
==> Extracting sources...
==> Sources are ready.
==> Making package: slack-desktop 3.3.4-1 (Fri 11 Jan 2019 05:45:23 PM CET)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> WARNING: Using existing $srcdir/ tree
==> Entering fakeroot environment...
==> Starting package()...
patching file usr/share/applications/slack.desktop
==> Tidying install...
  -> Removing libtool files...
  -> Purging unwanted files...
  -> Removing static library files...
  -> Stripping unneeded symbols from binaries and libraries...
  -> Compressing man and info pages...
==> Checking for packaging issues...
==> Creating package "slack-desktop"...
  -> Generating .PKGINFO file...
  -> Generating .BUILDINFO file...
  -> Generating .MTREE file...
  -> Compressing package...
==> Leaving fakeroot environment.
==> Finished making: slack-desktop 3.3.4-1 (Fri 11 Jan 2019 05:45:48 PM CET)
==> Cleaning up...
loading packages...
resolving dependencies...
looking for conflicting packages...

Packages (1) slack-desktop-3.3.4-1

Total Installed Size:  184.03 MiB
Net Upgrade Size:        3.27 MiB

:: Proceed with installation? [Y/n] 
(1/1) checking keys in keyring                     [######################] 100%
(1/1) checking package integrity                   [######################] 100%
(1/1) loading package files                        [######################] 100%
(1/1) checking for file conflicts                  [######################] 100%
(1/1) checking available disk space                [######################] 100%
:: Processing package changes...
(1/1) upgrading slack-desktop                      [######################] 100%
:: Running post-transaction hooks...
(1/2) Arming ConditionNeedsUpdate...
(2/2) Updating the desktop file MIME type cache...
==> Making package: clion 1:2018.3.3-1 (Fri 11 Jan 2019 05:45:53 PM CET)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Found CLion-2018.3.3.tar.gz
  -> Found jetbrains-clion.desktop
==> Validating source files with sha256sums...
    CLion-2018.3.3.tar.gz ... Passed
    jetbrains-clion.desktop ... Passed
==> Removing existing $srcdir/ directory...
==> Extracting sources...
==> Sources are ready.
==> Making package: clion 1:2018.3.3-1 (Fri 11 Jan 2019 05:45:57 PM CET)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> WARNING: Using existing $srcdir/ tree
==> Starting build()...
==> Entering fakeroot environment...
==> Starting package_clion()...
==> Tidying install...
  -> Removing libtool files...
  -> Purging unwanted files...
  -> Removing static library files...
  -> Compressing man and info pages...
==> Checking for packaging issues...
==> Creating package "clion"...
  -> Generating .PKGINFO file...
  -> Generating .BUILDINFO file...
  -> Generating .MTREE file...
  -> Compressing package...
==> Starting package_clion-jre()...
==> Tidying install...
  -> Removing libtool files...
  -> Purging unwanted files...
  -> Removing static library files...
  -> Compressing man and info pages...
==> Checking for packaging issues...
==> Creating package "clion-jre"...
  -> Generating .PKGINFO file...
  -> Generating .BUILDINFO file...
  -> Generating .MTREE file...
  -> Compressing package...
==> Starting package_clion-cmake()...
==> Tidying install...
  -> Removing libtool files...
  -> Purging unwanted files...
  -> Removing static library files...
  -> Compressing man and info pages...
==> Checking for packaging issues...
==> Creating package "clion-cmake"...
  -> Generating .PKGINFO file...
  -> Generating .BUILDINFO file...
  -> Generating .MTREE file...
  -> Compressing package...
==> Starting package_clion-gdb()...
==> Tidying install...
  -> Removing libtool files...
  -> Purging unwanted files...
  -> Removing static library files...
  -> Compressing man and info pages...
==> Checking for packaging issues...
==> Creating package "clion-gdb"...
  -> Generating .PKGINFO file...
  -> Generating .BUILDINFO file...
  -> Generating .MTREE file...
  -> Compressing package...
==> Starting package_clion-lldb()...
==> Tidying install...
  -> Removing libtool files...
  -> Purging unwanted files...
  -> Removing static library files...
  -> Compressing man and info pages...
==> Checking for packaging issues...
==> Creating package "clion-lldb"...
  -> Generating .PKGINFO file...
  -> Generating .BUILDINFO file...
  -> Generating .MTREE file...
  -> Compressing package...
==> Leaving fakeroot environment.
==> Finished making: clion 1:2018.3.3-1 (Fri 11 Jan 2019 05:48:35 PM CET)
==> Cleaning up...
loading packages...
resolving dependencies...
looking for conflicting packages...

Packages (3) clion-1:2018.3.3-1  clion-gdb-1:2018.3.3-1
             clion-lldb-1:2018.3.3-1

Total Installed Size:  760.78 MiB
Net Upgrade Size:        2.73 MiB

:: Proceed with installation? [Y/n] 
(3/3) checking keys in keyring                     [######################] 100%
(3/3) checking package integrity                   [######################] 100%
(3/3) loading package files                        [######################] 100%
(3/3) checking for file conflicts                  [######################] 100%
(3/3) checking available disk space                [######################] 100%
:: Processing package changes...
(1/3) upgrading clion                              [######################] 100%
(2/3) upgrading clion-gdb                          [######################] 100%
(3/3) upgrading clion-lldb                         [######################] 100%
:: Running post-transaction hooks...
(1/2) Arming ConditionNeedsUpdate...
(2/2) Updating the desktop file MIME type cache...
panic: runtime error: makeslice: cap out of range

goroutine 1 [running]:
main.(*depSolver).getMake(0xc000094820, 0xc0000cf950, 0x2, 0x2)
	/home/theaifam5/.cache/yay/yay/src/yay-9.0.1/depSolver.go:614 +0x73
main.install(0xc000020240, 0xc0000cfdc0, 0x2)
	/home/theaifam5/.cache/yay/yay/src/yay-9.0.1/install.go:323 +0x1a90
main.handleSync(0x9809f3, 0x1)
	/home/theaifam5/.cache/yay/yay/src/yay-9.0.1/cmd.go:271 +0x4a6
main.handleCmd(0x0, 0x0)
	/home/theaifam5/.cache/yay/yay/src/yay-9.0.1/cmd.go:153 +0x2f8
main.main()
	/home/theaifam5/.cache/yay/yay/src/yay-9.0.1/main.go:222 +0xd3
 ✘ theaifam5@pussy-master  ~ 
```

Config:
```
{
	"aururl": "https://aur.archlinux.org",
	"buildDir": "/home/theaifam5/.cache/yay",
	"editor": "",
	"editorflags": "",
	"makepkgbin": "makepkg",
	"makepkgconf": "",
	"pacmanbin": "pacman",
	"pacmanconf": "/etc/pacman.conf",
	"tarbin": "bsdtar",
	"redownload": "no",
	"rebuild": "no",
	"answerclean": "",
	"answerdiff": "",
	"answeredit": "",
	"answerupgrade": "",
	"gitbin": "git",
	"gpgbin": "gpg",
	"gpgflags": "",
	"mflags": "",
	"sortby": "votes",
	"gitflags": "",
	"removemake": "ask",
	"requestsplitn": 150,
	"sortmode": 0,
	"completionrefreshtime": 7,
	"sudoloop": false,
	"timeupdate": false,
	"devel": false,
	"cleanAfter": false,
	"gitclone": true,
	"provides": true,
	"pgpfetch": true,
	"upgrademenu": true,
	"cleanmenu": true,
	"diffmenu": false,
	"editmenu": true,
	"combinedupgrade": false,
	"useask": false
}
```

I had a crash as well at end of building and install.
```
=> Finished making: zdoom 2.8.1-5 (Sun 10 Feb 2019 10:00:48 AM JST)
==> Cleaning up...
loading packages...
warning: zdoom-2.8.1-5 is up to date -- reinstalling
resolving dependencies...
looking for conflicting packages...

Package (1)  Old Version  New Version  Net Change

zdoom        2.8.1-5      2.8.1-5        0.00 MiB

Total Installed Size:  7.39 MiB
Net Upgrade Size:      0.00 MiB

:: Proceed with installation? [Y/n] 
(1/1) checking keys in keyring                                         [----------------------------------------] 100%
(1/1) checking package integrity                                       [----------------------------------------] 100%
(1/1) loading package files                                            [----------------------------------------] 100%
(1/1) checking for file conflicts                                      [----------------------------------------] 100%
(1/1) checking available disk space                                    [----------------------------------------] 100%
:: Processing package changes...
(1/1) reinstalling zdoom                                               [----------------------------------------] 100%
:: Running post-transaction hooks...
(1/3) Updating icon theme caches...
(2/3) Arming ConditionNeedsUpdate...
(3/3) Updating the desktop file MIME type cache...
panic: runtime error: makeslice: cap out of range

goroutine 1 [running]:
main.(*depOrder).getMake(0xc0000a2100, 0xc0000e5950, 0x2, 0x2)
	/build/src/yay-9.1.0/depOrder.go:118 +0x73
main.install(0xc0000a2180, 0xc0000e5dc0, 0x2)
	/build/src/yay-9.1.0/install.go:329 +0x1999
main.handleSync(0x9819f3, 0x1)
	/build/src/yay-9.1.0/cmd.go:274 +0x46c
main.handleCmd(0x0, 0x0)
	/build/src/yay-9.1.0/cmd.go:153 +0x2f8
main.main()
	/build/src/yay-9.1.0/main.go:222 +0xd3

```

I got the same issue. @Morganamilo @Jguer 

```
$ yay --version
yay v9.1.0 - libalpm v11.0.3
```
Crash Log
```
:: Proceed with installation? [Y/n] 
(2/2) checking keys in keyring                                                                                     [#####################################################################] 100%
(2/2) checking package integrity                                                                                   [#####################################################################] 100%
(2/2) loading package files                                                                                        [#####################################################################] 100%
(2/2) checking for file conflicts                                                                                  [#####################################################################] 100%
(2/2) checking available disk space                                                                                [#####################################################################] 100%
:: Processing package changes...
(1/2) upgrading webstorm                                                                                           [#####################################################################] 100%
(2/2) upgrading webstorm-jre                                                                                       [#####################################################################] 100%
:: Running post-transaction hooks...
(1/2) Arming ConditionNeedsUpdate...
(2/2) Updating the desktop file MIME type cache...
panic: runtime error: makeslice: cap out of range

goroutine 1 [running]:
main.(*depOrder).getMake(0xc0000249c0, 0xc000185c20, 0x2, 0x2)
        /home/xuan/yay/src/yay-9.1.0/depOrder.go:118 +0x78
main.install(0xc0000ac380, 0xc0000d5e50, 0x2)
        /home/xuan/yay/src/yay-9.1.0/install.go:329 +0x1162
main.handleSync(0xa13bd3, 0x1)
        /home/xuan/yay/src/yay-9.1.0/cmd.go:271 +0x449
main.handleCmd(0x0, 0x0)
        /home/xuan/yay/src/yay-9.1.0/cmd.go:153 +0x384
main.main()
        /home/xuan/yay/src/yay-9.1.0/main.go:222 +0xec
```

The same problem happened again in yay 9.2.1.

- yay version: `yay v9.2.1 - libalpm v11.0.3`
- Distribution: Archlinux
- Go version: `go version go1.13 linux/amd64`

Crash log & package list:
```
:: Starting full system upgrade...
resolving dependencies...
looking for conflicting packages...

Package (126)                           Old Version           New Version           Net Change 

extra/adwaita-icon-theme                3.32.0-1              3.34.0-1                 0.14 MiB
extra/akonadi                           19.08.1-1             19.08.1-2                0.00 MiB
community/android-tools                 29.0.2-2              29.0.3-1                -0.05 MiB
extra/at-spi2-atk                       2.32.0-1              2.34.0-1                 0.00 MiB
extra/at-spi2-core                      2.32.1-1              2.34.0-1                 0.03 MiB
extra/atk                               2.32-1                2.34.1-1                 0.08 MiB
community/bmake                                               20181221-1               0.51 MiB
extra/bubblewrap                                              0.3.3-3                  0.06 MiB
extra/chromium                          76.0.3809.132-1       77.0.3865.75-1           2.24 MiB
extra/cmake                             3.15.3-1              3.15.3-2                 0.28 MiB
core/cryptsetup                         2.2.0-1               2.2.1-1                  0.05 MiB
community/cudnn                         7.6.2.24-1            7.6.3.30-1              -3.65 MiB
core/curl                               7.65.3-1              7.66.0-1                 0.01 MiB
extra/dconf                             0.32.0-3              0.34.0-1                 0.00 MiB
core/device-mapper                      2.02.186-1            2.02.186-2               0.00 MiB
extra/devtools                          20190821-1            20190912-1               0.00 MiB
community/electron                      6.0.7-1               6.0.9-1                  0.00 MiB
core/fakeroot                           1.23-1                1.24-1                   0.01 MiB
community/fcitx-qt5                     1.2.3-9               1.2.3-10                 0.00 MiB
community/gammaray                      2.11.0-1              2.11.0-2                -0.02 MiB
extra/gcr                               3.28.1-1              3.33.4+6+g35df18d-1      0.25 MiB
extra/gdk-pixbuf2                       2.38.1-1              2.38.2-1                 0.05 MiB
core/gettext                            0.20.1-1              0.20.1-2                 0.00 MiB
extra/glib-networking                   2.60.3-1              2.62.0-1                 0.01 MiB
core/glib2                              2.60.7-1              2.62.0-1                 0.14 MiB
extra/glslang                           7.11.3214-2           7.12.3352-1              0.13 MiB
extra/gobject-introspection-runtime     1.60.2-1              1.62.0-1                 0.06 MiB
extra/graphene                          1.8.6-1               1.10.0-1                 0.33 MiB
extra/graphviz                          2.40.1-19             2.42.1-1                 0.93 MiB
extra/gsettings-desktop-schemas         3.32.0-2              3.34.0-1                 0.11 MiB
extra/hspell                            1.4-1                 1.4-2                    0.01 MiB
community/hub                           2.12.3-3              2.12.4-1                 0.12 MiB
extra/imagemagick                       7.0.8.63-1            7.0.8.64-1               0.00 MiB
extra/jsoncpp                           1.8.4-2               1.9.1-1                  0.07 MiB
extra/kwin                              5.16.5-1              5.16.5-2                 0.00 MiB
multilib/lib32-atk                      2.32.0-1              2.34.1-1                 0.00 MiB
multilib/lib32-gdk-pixbuf2              2.38.1-1              2.38.2-1                 0.00 MiB
multilib/lib32-glib2                    2.60.7-1              2.62.0-1                 0.04 MiB
multilib/lib32-libpulse                 12.2-1                12.99.3-1                0.00 MiB
multilib/lib32-openssl                  1:1.1.1.c-1           1:1.1.1.d-1              0.00 MiB
multilib/lib32-p11-kit                  0.23.16.1-1           0.23.17-2               -0.38 MiB
multilib/lib32-systemd                  242.84-1              243.0-1                  0.02 MiB
extra/libakonadi                        19.08.1-1             19.08.1-2                0.00 MiB
community/libcurl-compat                7.65.3-1              7.66.0-1                 0.00 MiB
community/libcurl-gnutls                7.65.3-1              7.66.0-1                 0.00 MiB
extra/libical                           3.0.4-3               3.0.5-1                  4.92 MiB
extra/libjpeg-turbo                     2.0.2-1               2.0.3-1                  0.01 MiB
extra/libmagick6                        6.9.10.63-1           6.9.10.64-1              0.00 MiB
extra/libmbim                           1.18.2-1              1.20.0-1                 0.00 MiB
extra/libmm-glib                        1.10.4-1              1.10.6-1                 0.14 MiB
core/libnftnl                           1.1.3-1               1.1.4-1                  0.01 MiB
extra/libplacebo                        1.18.0-1              1.18.0-2                -0.05 MiB
extra/libpulse                          12.2-2                12.99.3-1                0.04 MiB
extra/libqmi                            1.22.4-1              1.22.6-1                 0.04 MiB
extra/librsvg                           2:2.44.15-1           2:2.46.0-1              -1.18 MiB
core/libsecret                          0.18.8-2              0.19.1-1                 0.24 MiB
extra/libsoup                           2.66.2+4+g716acf96-1  2.68.1-1                 0.40 MiB
extra/libstemmer                        0+337-3               0+713-1                  0.24 MiB
extra/libwpe                                                  1.3.91-1                 0.26 MiB
core/linux                              5.2.11.arch1-1        5.2.14.arch2-1           0.01 MiB
core/linux-headers                      5.2.11.arch1-1        5.2.14.arch2-1           0.00 MiB
core/logrotate                          3.15.0-1              3.15.1-1                 0.00 MiB
extra/lsof                              4.91-1                4.93.2-1                 0.00 MiB
core/lvm2                               2.02.186-1            2.02.186-2               0.00 MiB
extra/mariadb                           10.4.7-1              10.4.8-2                 0.45 MiB
extra/mariadb-clients                   10.4.7-1              10.4.8-2                 0.00 MiB
extra/mariadb-libs                      10.4.7-1              10.4.8-2                 0.05 MiB
extra/mesa                              19.1.6-1              19.1.6-2                -0.08 MiB
extra/modemmanager                      1.10.4-1              1.10.6-1                 0.01 MiB
extra/mpg123                            1.25.11-1             1.25.12-1                0.04 MiB
community/neofetch                      6.0.0-1               6.1.0-1                  0.01 MiB
extra/nvidia                            435.21-1              435.21-4                 0.00 MiB
extra/openconnect                       1:8.04-1              1:8.05-1                 0.02 MiB
core/openssl                            1.1.1.c-1             1.1.1.d-1               -0.01 MiB
extra/orc                               0.4.29-1              0.4.30-1                 0.01 MiB
core/p11-kit                            0.23.16.1-1           0.23.17-2               -0.11 MiB
extra/perl-alien-build                  1.85-1                1.86-1                   0.00 MiB
extra/pulseaudio                        12.2-2                12.99.3-1                0.37 MiB
extra/pulseaudio-bluetooth              12.2-2                12.99.3-1                0.02 MiB
extra/pygobject-devel                   3.32.2-1              3.34.0-1                 0.00 MiB
extra/python-gobject                    3.32.2-1              3.34.0-1                 0.00 MiB
community/python-importlib-metadata     0.20-1                0.22-1                   0.01 MiB
extra/qt5-3d                            5.13.0-1              5.13.1-1                 0.02 MiB
extra/qt5-base                          5.13.0-7              5.13.1-2                 0.30 MiB
extra/qt5-declarative                   5.13.0-1              5.13.1-1                 0.09 MiB
extra/qt5-doc                           5.13.0-1              5.13.1-1              -142.42 MiB
extra/qt5-examples                      5.13.0-1              5.13.1-1                -3.29 MiB
extra/qt5-graphicaleffects              5.13.0-1              5.13.1-1                 0.00 MiB
extra/qt5-imageformats                  5.13.0-2              5.13.1-1                 0.00 MiB
extra/qt5-location                      5.13.0-1              5.13.1-1                 0.00 MiB
extra/qt5-multimedia                    5.13.0-1              5.13.1-1                 0.01 MiB
extra/qt5-networkauth                   5.13.0-1              5.13.1-1                 0.00 MiB
extra/qt5-quickcontrols                 5.13.0-1              5.13.1-1                -0.02 MiB
extra/qt5-quickcontrols2                5.13.0-1              5.13.1-1                 0.00 MiB
extra/qt5-script                        5.13.0-1              5.13.1-1                 0.00 MiB
extra/qt5-sensors                       5.13.0-1              5.13.1-1                 0.01 MiB
extra/qt5-speech                        5.13.0-1              5.13.1-1                 0.00 MiB
extra/qt5-svg                           5.13.0-1              5.13.1-1                 0.00 MiB
extra/qt5-tools                         5.13.0-2              5.13.1-1                 0.03 MiB
extra/qt5-translations                  5.13.0-1              5.13.1-1                 0.00 MiB
extra/qt5-wayland                       5.13.0-3              5.13.1-1                 0.00 MiB
extra/qt5-webchannel                    5.13.0-1              5.13.1-1                 0.01 MiB
extra/qt5-webengine                     5.13.0-4              5.13.1-2                 0.08 MiB
extra/qt5-webkit                        5.212.0alpha3-3       5.212.0alpha3-4          0.00 MiB
extra/qt5-x11extras                     5.13.0-1              5.13.1-1                 0.00 MiB
extra/qt5-xmlpatterns                   5.13.0-1              5.13.1-1                 0.00 MiB
extra/qtcreator                         4.9.2-3               4.10.0-1                 2.90 MiB
extra/rpcsvc-proto                                            1.4-1                    0.20 MiB
community/rustup                        1.18.3-1              1.19.0-1                 0.24 MiB
extra/shaderc                           2019.0-1              2019.0-2                 0.00 MiB
extra/srt                               1.3.4-1               1.4.0-1                  0.97 MiB
community/syncthing                     1.2.1-2               1.2.2-1                  6.58 MiB
core/systemd                            242.84-2              243.0-1                  0.78 MiB
core/systemd-libs                       242.84-2              243.0-1                  0.01 MiB
core/systemd-sysvcompat                 242.84-2              243.0-1                  0.00 MiB
core/tzdata                             2019b-1               2019c-1                  0.00 MiB
extra/upower                            0.99.10-1             0.99.11-1                0.00 MiB
community/virtualbox-host-modules-arch  6.0.12-1              6.0.12-4                 0.00 MiB
extra/webkit2gtk                        2.24.4-1              2.26.0-1                 3.21 MiB
community/winetricks                    20190615-1            20190912-1               0.00 MiB
community/wireshark-cli                 3.0.3-1               3.0.4-1                  0.04 MiB
community/wireshark-qt                  3.0.3-1               3.0.4-1                  0.01 MiB
extra/wpebackend-fdo                                          1.3.91-1                 0.07 MiB
extra/xdg-dbus-proxy                                          0.1.1-1                  0.04 MiB
extra/zenity                            3.32.0-1              3.32.0+17+gfb0938f-1     0.11 MiB
community/zeromq                        4.3.1-2               4.3.2-1                  0.10 MiB

Total Installed Size:  3320.25 MiB
Net Upgrade Size:      -122.46 MiB

:: Proceed with installation? [Y/n] 
(126/126) checking keys in keyring
(126/126) checking package integrity
(126/126) loading package files
(126/126) checking for file conflicts
(126/126) checking available disk space
:: Running pre-transaction hooks...
(1/1) Remove DKMS modules
==> dkms remove exfat/205.c0915bb -k 5.2.11-arch1-1-ARCH
:: Processing package changes...

...(omitted for brevity)

:: Running post-transaction hooks...
( 1/21) Creating system user accounts...
( 2/21) Updating journal message catalog...
( 3/21) Reloading system manager configuration...
( 4/21) Updating udev hardware database...
( 5/21) Applying kernel sysctl settings...
( 6/21) Creating temporary files...
( 7/21) Reloading device manager configuration...
( 8/21) Arming ConditionNeedsUpdate...
( 9/21) Updating linux module dependencies...
(10/21) Install DKMS modules
==> dkms install exfat/205.c0915bb -k 5.2.14-arch2-1-ARCH
(11/21) Updating linux initcpios...

...(omitted for brevity)

(12/21) Reloading system bus configuration...
(13/21) Warn about old perl modules
WARNING: '/usr/lib/perl5/5.28' contains data from at least 1 packages which will NOT be used by the installed perl interpreter.
 -> Run the following command to get a list of affected packages: pacman -Qqo '/usr/lib/perl5/5.28'
(14/21) Probing 32-bit GDK-Pixbuf loader modules...
(15/21) Probing GDK-Pixbuf loader modules...
(16/21) Updating GIO module cache...
(17/21) Compiling GSettings XML schema files...
(18/21) Updating icon theme caches...
(19/21) Updating the info directory file...
(20/21) Updating the desktop file MIME type cache...
(21/21) Updating the MIME type database...
[sudo] password for user: 
removing Untracked AUR files from cache...
:: Cleaning (1/9): /home/user/.cache/yay/libnvidia-container
:: Cleaning (2/9): /home/user/.cache/yay/nvidia-container-toolkit
:: Cleaning (3/9): /home/user/.cache/yay/nvidia-container-runtime
:: Cleaning (4/9): /home/user/.cache/yay/nvidia-docker
:: Cleaning (5/9): /home/user/.cache/yay/clion
:: Cleaning (6/9): /home/user/.cache/yay/jupyter_contrib_nbextensions
:: Cleaning (7/9): /home/user/.cache/yay/pycharm-professional
:: Cleaning (8/9): /home/user/.cache/yay/visual-studio-code-bin
:: Cleaning (9/9): /home/user/.cache/yay/gitkraken
panic: runtime error: makeslice: cap out of range

goroutine 1 [running]:
main.(*depOrder).getMake(0xc000080100, 0xc0001a17a8, 0x2, 0x2)
        depOrder.go:118 +0x78
main.removeMake(0xc000080100, 0xc0001a1e18)
        install.go:337 +0x13f
main.install(0xc0000c4380, 0x800900, 0xc000204010)
        install.go:306 +0x1d69
main.handleSync(0xa14bf3, 0x1)
        cmd.go:272 +0x449
main.handleCmd(0x0, 0x0)
        cmd.go:154 +0x384
main.main()
        main.go:223 +0xec
```

I just got the same message.

```
-- Installing: /home/kodehawa/.cache/yay/python-pyside/pkg/python-pyside/usr/share/PySide/typesystems/typesystem_declarative.xml
==> Tidying install...
  -> Removing libtool files...
  -> Purging unwanted files...
  -> Removing static library files...
  -> Stripping unneeded symbols from binaries and libraries...
  -> Compressing man and info pages...
==> Checking for packaging issues...
==> WARNING: Package contains reference to $srcdir
usr/lib/libpyside.cpython-37m-x86_64-linux-gnu.so.1.2.4
usr/lib/python3.7/site-packages/PySide/QtNetwork.so
usr/lib/python3.7/site-packages/PySide/QtGui.so
usr/lib/python3.7/site-packages/PySide/QtMultimedia.so
usr/lib/python3.7/site-packages/PySide/QtXml.so
usr/lib/python3.7/site-packages/PySide/QtDeclarative.so
usr/lib/python3.7/site-packages/PySide/QtOpenGL.so
usr/lib/python3.7/site-packages/PySide/phonon.so
usr/lib/python3.7/site-packages/PySide/QtSql.so
usr/lib/python3.7/site-packages/PySide/QtScriptTools.so
usr/lib/python3.7/site-packages/PySide/QtCore.so
usr/lib/python3.7/site-packages/PySide/QtXmlPatterns.so
usr/lib/python3.7/site-packages/PySide/QtSvg.so
usr/lib/python3.7/site-packages/PySide/QtTest.so
usr/lib/python3.7/site-packages/PySide/QtScript.so
usr/lib/python3.7/site-packages/PySide/QtHelp.so
usr/lib/python3.7/site-packages/PySide/QtUiTools.so
==> Creating package "python-pyside"...
  -> Generating .PKGINFO file...
  -> Generating .BUILDINFO file...
  -> Generating .MTREE file...
  -> Compressing package...
==> Leaving fakeroot environment.
==> Finished making: python-pyside 1.2.4-9 (Sun 22 Sep 2019 04:01:26 PM -03)
==> Cleaning up...
[sudo] password for kodehawa:             
loading packages...
resolving dependencies...
looking for conflicting packages...

Packages (2) python-pyside-1.2.4-9  python-pyside-common-1.2.4-9

Total Installed Size:  22.87 MiB

:: Proceed with installation? [Y/n] 
(2/2) checking keys in keyring                         [############################] 100%
(2/2) checking package integrity                       [############################] 100%
(2/2) loading package files                            [############################] 100%
(2/2) checking for file conflicts                      [############################] 100%
(2/2) checking available disk space                    [############################] 100%
:: Processing package changes...
(1/2) installing python-pyside-common                  [############################] 100%
(2/2) installing python-pyside                         [############################] 100%
Optional dependencies for python-pyside
    phonon-qt4: for PySide.phonon [installed]
:: Running post-transaction hooks...
(1/1) Arming ConditionNeedsUpdate...
panic: runtime error: makeslice: cap out of range

goroutine 1 [running]:
main.(*depOrder).getMake(0xc000654000, 0xc00017f288, 0x2, 0x2)
        depOrder.go:118 +0x78
main.removeMake(0xc000654000, 0xc00017f8a8)
        install.go:360 +0x13f
main.install(0xc00006e200, 0x0, 0x0)
        install.go:353 +0x10b5
main.displayNumberMenu(0xc00005b4a0, 0x1, 0x1, 0x2, 0x0)
        cmd.go:399 +0x7c9
main.handleYogurt(...)
        cmd.go:242
main.handleYay(0x79d688, 0x1)
        cmd.go:231 +0x150
main.handleCmd(0x0, 0x0)
        cmd.go:164 +0x2b5
main.main()
        main.go:223 +0xeb
```

--- a/depOrder.go
+++ b/depOrder.go
@@ -115,7 +115,7 @@ func (do *depOrder) HasMake() bool {
 }
 
 func (do *depOrder) getMake() []string {
-	makeOnly := make([]string, 0, len(do.Aur)+len(do.Repo)-len(do.Runtime))
+	makeOnly := []string{}
 
 	for _, base := range do.Aur {
 		for _, pkg := range base {

